MbDotNet/Models/Requests/SmtpRequest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MbDotNet.Models.Requests
+{
+    public class SmtpRequest: Request
+    {
+        [JsonProperty("envelopeFrom")]
+        public string EnvelopeFrom { get; internal set; }
+
+        [JsonProperty("envelopeTo")]
+        public IList<string> EnvelopeTo { get; internal set; }
```

Comment:
According to the docs, this is a single string, not a collection.

Reply:
Please see above